documentation/IDTA-01001/modules/ROOT/pages/mappings/encodings/idshortpath.adoc

```diff
@@ -18,6 +18,10 @@ In the first case, the submodel element's xref:spec-metamodel/common.adoc#Refera
 In the second case, square brackets with an index "[<Index>]" are appended after the idShort of the containing SubmodelElementList.
 In any case, the first item of any idShortPath is the idShort of the requested element.
 
+====
+Note: Although idShort may be defined for elements within a SubmodelElementList the index shall be used within the IdShortPath serialization, only
```

Comment:
This is gramatically imprecise, as it suggests that the index shall only ever be used in the SubmodelElementList and nowhere else. I think it should be: 

```suggestion
Note: Although idShort may be defined for elements within a SubmodelElementList, only the index shall be used within the IdShortPath serialization.
```